apps/judicial-system/backend/src/app/modules/subpoena/guards/subpoenaExists.guard.ts

```diff
@@ -3,34 +3,28 @@ import {
   CanActivate,
   ExecutionContext,
   Injectable,
+  InternalServerErrorException,
 } from '@nestjs/common'
 
 import { Defendant } from '../../defendant'
-import { SubpoenaService } from '../subpoena.service'
 
 @Injectable()
 export class SubpoenaExistsGuard implements CanActivate {
```

Comment:
The subpoena guard has been split in two. This guard handles retrieving subpoens by the internal subpoena id.